### PR TITLE
chore(html): reword admonition for clarity on order of defining img attributes

### PR DESCRIPTION
**Pull request by bsmth**
Follow-up after PR comments in https://github.com/mdn/content/pull/26778#issuecomment-1688116795

### CODE REVIEW

File: files/en-us/web/html/element/img/index.md
Comment (bsmth) [inline files/en-us/web/html/element/img/index.md:1]: Note that the API might be the appropriate place for this instead: https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading

**Comment by schardev** [comment]
> "... `loading` attribute must be defined (i.e. in JavaScript) before the `src` attribute ..."

Although this sounds a little vague (or maybe I'm just over-evaluating), the linked bug report should provide sufficient clarity.

This looks fine to me, but maybe ask others if it looks clear enough to them.

File: files/en-us/web/html/element/img/index.md
Comment (Josh-Cena) [reply files/en-us/web/html/element/img/index.md:1]: I agree. We should move it to the API side instead, because this has nothing to do with HTML.

**Review by estelle — APPROVED** [review]

**Review by estelle — APPROVED** [review]
you might consider adding a link to the API in the img element's see also section.

**Pull request merged**